lib/models/messages/twitch/eventsub_configuration.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:flutter/material.dart';
 
 class FollowEventConfig {
-  final bool showEvent;
-  final bool isEventPinnable;
-  final int eventDuration;
+  bool showEvent;
+  bool isEventPinnable;
+  double eventDuration;
```

Comment:
Dart also has a `Duration` type you can use here: https://api.dart.dev/stable/2.13.4/dart-core/Duration-class.html

Reply:
changed the type from `double` to `Duration`.